File: android/src/main/java/name/zeno/android/tint/TintableCompoundDrawableView.java
package name.zeno.android.tint;

import android.content.res.ColorStateList;
import android.graphics.PorterDuff;
import android.support.annotation.Nullable;

/**
 * Create Date: 16/6/23
 *
 * @author 陈治谋 (513500085@qq.com)
 */
public interface TintableCompoundDrawableView
{

  void setSupportCompoundDrawableTintList(@Nullable ColorStateList tint);

  void setSupportCompoundDrawableTintMode(@Nullable PorterDuff.Mode tintMode);

  @Nullable ColorStateList getSupportCompoundDrawableTintList();

  @Nullable PorterDuff.Mode getSupportCompoundDrawableTintMode();
}

File: library/src/main/java/name/zeno/android/tint/TintableCompoundDrawableView.kt
package name.zeno.android.tint

import android.content.res.ColorStateList
import android.graphics.PorterDuff

/**
 * Create Date: 16/6/23
 *
 * @author 陈治谋 (513500085@qq.com)
 */
interface TintableCompoundDrawableView {

  var supportCompoundDrawableTintList: ColorStateList?

  var supportCompoundDrawableTintMode: PorterDuff.Mode?
}
